langchain/vectorstores/elastic_vector_search.py

```diff
@@ -113,25 +127,41 @@ class ElasticVectorSearch(VectorStore, ABC):
     Raises:
         ValueError: If the elasticsearch python package is not installed.
     """
-
-    def __init__(self, elasticsearch_url: str, index_name: str, embedding: Embeddings):
+    def __init__(self, **kw):
         """Initialize with necessary components."""
-        try:
-            import elasticsearch
-        except ImportError:
+        elasticsearch_url = kw.get("elasticsearch_url", None)
+        elasticsearch_client = kw.get("elasticsearch_client", None)
```

Comment:
is this only parameter we're adding? can we just add this as an explicit arg/kwarg, do we need to convert to using `**kw`?